packages/cspell/samples/fail-fast/fail-fast-cspell.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+    // Version of the setting file.  Always 0.1
```

Comment:
Please remove the comments. They are not right anyway. (I know it is a copy :-) )

Reply:
Yes, it was a copy :) Also, to reduce duplication a bit, I used `import`